app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/test/chapter4/conditionwatchers/ConditionWatchers.kt

```diff
@@ -90,6 +93,33 @@ object ConditionWatchers {
         return interaction
     }
 
+    /*
+    Helper method that can be used with the DataInteraction type.
+     */
+
+    fun waitForElement(
+            interaction: DataInteraction,
```

Comment:
Indentation fo those first 4 lines seems a bit odd.

Reply:
How can I change it?  I used the ViewInteraction waitForElement() function as an example.